### PR TITLE
Uploading with full paths

### PR DESCRIPTION
**Pull request by bnookala**
CLI uploading uses user folders.

### CODE REVIEW

File: cli/operations.py [hunk 1]
```diff
@@ -17,62 +17,78 @@
...
+    if len(onboarding_files) == 0:
+        print(f'Could not find any valid files to upload')
+        return
+
+    for blob_path in onboarding_files:
+        stripped_path = strip_path_prefix(folder_name, blob_path)
+        print("Uploading " + str(blob_path))
 
-        # Upload the created file, use image name for the blob name
         blob_storage.create_blob_from_path(
```
Comment (andrebriggs) [inline cli/operations.py:87]: @bnookala if we are making changes here how about adding blob metadata like [here] (https://github.com/CatalystCode/active-learning-detect/blob/a8a992231786fdaa43384119999376676b87dbcc/functions/pipeline/onboardqueueproccessor/__init__.py#L83) ?

File: cli/operations.py [hunk 2]
```diff
@@ -17,62 +17,78 @@
...
+        return False
+
     file_suffix = pathlib.Path(file_name).suffix.lower()
     if file_suffix in ['.png', '.jpg', '.jpeg', '.gif']:
         return True
     else:
         return False
 
 
+# Somewhat of a hack to remove the folder name passed on the
```
Comment (andrebriggs) [inline cli/operations.py:36]: @bnookala what does an example of this output look like?

Comment (bnookala) [reply cli/operations.py:36]: @andrebriggs I added this to the code file as documentation, but here it is again for posterity:
```
# Example invocation: python3 -m cli onboard /my/full/path
# os.walk will return: /my/full/path/1.jpg, /my/full/path/2.jpg, etc.
```

File: cli/operations.py (hunk 1 above)
Comment (bnookala) [reply cli/operations.py:87]: done

**Comment by bnookala** [comment]
@andrebriggs @mtarng please take a look at this!

**Review by mtarng — APPROVED** [review]

**Pull request merged**